Probesys/bileto
# new: Send a notification when posting a new message

## Related issue(s)



https://github.com/Probesys/bileto/issues/330

## Changes



- add and configure the Symfony Mailer et Notifier components
- create a NewMessageNotification and send it when a message is posted

## How to test manually



- open a ticket and assign a user different from you
- post a new message (not confidential!)
- follow the instructions from GreenMail documentation (under docs/developers) to configure an external mail client with the mailbox of the other user
- check the email arrived in the mailbox

## Checklist
